packages/fields/src/components/create-template-part-modal/index.tsx

```diff
@@ -203,51 +202,60 @@ export function CreateTemplatePartModalContents( {
 					onChange={ setTitle }
 					required
 				/>
-				<BaseControl
-					__nextHasNoMarginBottom
-					label={ __( 'Area' ) }
-					id={ `fields-create-template-part-modal__area-selection-${ instanceId }` }
-					className="fields-create-template-part-modal__area-base-control"
-				>
-					<RadioGroup
-						label={ __( 'Area' ) }
-						className="fields-create-template-part-modal__area-radio-group"
-						id={ `fields-create-template-part-modal__area-selection-${ instanceId }` }
-						onChange={ ( value ) =>
-							value && typeof value === 'string'
-								? setArea( value )
-								: () => void 0
-						}
-						checked={ area }
+				<fieldset>
+					<BaseControl.VisualLabel
+						as="legend"
+						className="fields-create-template-part-modal__area-radio-group-label"
 					>
+						{ __( 'Area' ) }
+					</BaseControl.VisualLabel>
+					<div className="fields-create-template-part-modal__area-radio-group">
 						{ defaultTemplatePartAreas.map(
 							( { icon, label, area: value, description } ) => (
-								<Radio
-									__next40pxDefaultSize
-									key={ label }
-									value={ value }
-									className="fields-create-template-part-modal__area-radio"
+								<div
+									key={ value }
+									className="fields-create-template-part-modal__area-radio-wrapper"
 								>
-									<Flex align="start" justify="start">
-										<FlexItem>
-											<Icon icon={ icon } />
-										</FlexItem>
-										<FlexBlock className="fields-create-template-part-modal__option-label">
-											{ label }
-											<div>{ description }</div>
-										</FlexBlock>
-
-										<FlexItem className="fields-create-template-part-modal__checkbox">
-											{ area === value && (
-												<Icon icon={ check } />
-											) }
-										</FlexItem>
-									</Flex>
-								</Radio>
+									<input
+										type="radio"
+										id={ getAreaRadioId( value ) }
+										name="fields-create-template-part-modal__area"
```

Comment:
I think you meant `className` here, but also it looks unused?

Reply:
Shouldn't all radios in the same radio group share the same `name` ?

Reply:
Lol sorry, you're right. Same comment about maybe using an instance id applies then.

Reply:
https://github.com/WordPress/gutenberg/pull/67702/commits/16fd5321f39899713d7546d869d505ea9d23d6be